src/coreclr/nativeaot/Runtime/ObjectLayout.h

```diff
@@ -37,9 +37,13 @@ typedef DPTR(class EEType) PTR_EEType;
 class MethodTable;
 
 //-------------------------------------------------------------------------------------------------
+
+extern "C" void PopulateDebugHeaders();
```

Comment:
It doesn't look like the changes to this file are necessary anymore.

Reply:
Removed those too